core/platform/transactions/gae_transaction_services.py

```diff
@@ -48,6 +48,36 @@ def run_in_transaction(fn, *args, **kwargs):
     )
 
 
+def transaction_wrapper(fn, *args, **kwargs):
```

Comment:
Is this the approach you're recommending for all transactions in the codebase? We should have an issue tracking moving other uses over to this pattern so that we can have consistency across the backend.

Reply:
I actually lean pretty strongly toward standardizing one approach, otherwise it can result in confusion between why some operations use approach A vs B if the differences between A and B aren't really obvious (and they aren't here, I think). I think the decorator is a nice design, but in the effort to standardize can you file an issue tracking moving other transaction methods over to the decorator after this PR goes in? That seems like a nice starter issue to work on.

Reply:
Done.